screens/UserOptionsScreen/index.js

```diff
@@ -1,76 +1,57 @@
-import {useState, useEffect} from 'react';
-import PropTypes from 'prop-types';
-import {View, Text, TouchableOpacity, FlatList} from 'react-native';
-import {theme} from '../../assets/theme/theme';
-import Header from '../../components/Header/Header';
-import {getDataOptionsFromFile} from '../../utils/ApiUtils';
+import {memo, useState, useCallback} from 'react';
 import styles from './style';
+import {FlatList, Text, TouchableOpacity, View} from 'react-native';
+import {theme} from '../../assets/theme/theme';
+import useDataFromAPI from '../../customHooks/UseDataFromAPI';
 
-const UserOptionsScreen = ({navigation}) => {
-  const [data, setData] = useState([]);
+const OptionsList = () => {
   const [activated, setActivated] = useState(false);
 
-  useEffect(() => {
-    const fetchData = () => {
-      const result = getDataOptionsFromFile();
-      setData(result);
-    };
-    fetchData();
+  const {data} = useDataFromAPI('options');
+
+  const getBorderColor = useCallback(selected => {
+    return selected ? theme.colors.primary_green : theme.colors.cool_gray;
```

Comment:
there is no need to write inside useCallback move outside from it

Reply:
Done !